tests/unit_tests/analysis/test_es_update.py

```diff
@@ -206,7 +211,8 @@ def test_update_snapshot(
             w_container,
             "id",
             update_configuration,
-            ert_config.analysis_config,
+            UpdateSettings(),
+            IESSettings(ies_inversion=1),
```

Comment:
I fail to see why this is set to `1`. Default used to be `0` no? Does this maybe not affect the tests?

Reply:
Default is 0, yes, but in this config it has been set to 1:
```
ANALYSIS_SET_VAR IES_ENKF IES_INVERSION 1
ANALYSIS_SET_VAR STD_ENKF IES_INVERSION 1
```
can change it to:
```python
ert_config.analysis_config.es_module,
```
instead if you find that more readable?

Reply:
It does affect the tests

Reply:
No this is fine as is, I was just curious why this was specifically set to that value. And now you've clarified that.

Reply:
I retract my retraction, if set to something else it becomes flaky 😅